projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts

```diff
@@ -56,7 +56,7 @@ describe('AllowedRolesDirective', () => {
 				]
 		})
 			.compileComponents();
-	}));
+	});
 
 	beforeEach(() => {
```

Comment:
Do we need to put async on this beforeEach as we have on line 45? Thanks!

Reply:
The TestBed.compileComponents method asynchronously compiles all components configured in the testing module.
Please see https://angular.io/guide/testing-components-scenarios#the-async-beforeeach

Reply:
Thanks!